code/modules/mob/living/carbon/carbon.dm

```diff
@@ -851,7 +851,7 @@
 /mob/living/carbon/fully_heal(admin_revive = FALSE)
 	if(reagents)
 		reagents.clear_reagents()
-		for(var/addi in reagents.addiction_list)
+		for(var/addi in reagents?.addiction_list)
```

Comment:
And this wouldn't

Reply:
Probably going to revert this to a normal if statement.